profiles/templates/profiles/view_profile.html

```diff
@@ -15,4 +15,72 @@ <h3>Created Events:</h3>
+              </form>   
+          {% elif join_status == WITHDRAWN or join_status == None or join_status == REJECTED or join_status == REMOVED %}
+              <!-- Button to join again or create a new EventJoin object-->
+              <form action="{% url 'profiles:toggle-friend-request' user_profile.id %}" method="post">
+                  {% csrf_token %}
+                  <button type="submit" class="btn btn-primary btn-block join-button" id="request-to-join" onclick="return confirm('Ready to send a request to join?');">Send Friend Request</button>
+              </form>
+          {% elif join_status == APPROVED %}
+          <!-- show that the request is approved-->
+          <!-- <div class="btn btn-primary btn-block join-button" id="approved-fake">Friend Requests</div>   -->
+          {% endif %}
+          <div class="approved-join-list sticky-note-container my-4">
+              <h3>Friends</h3>
+              <div class="join-entry">{{ join.user }}</div>
+              {% for join in approved_join %}
+                  <div class="join-entry">
+                      {{ join.user }}</div>
+              {% endfor %}
+          </div>
+      {% else %}
+          <!-- The user is the creator of the event -->
+          <!-- <div class="btn btn-primary btn-block join-button" id="my-event-fake"></div> -->
+          <div class="pending-join-list sticky-note-container my-4">
+              <h3>Pending Friend Requests ({{ pending_join_count }})</h3>
+              {% for join in pending_join %}
+                  <div class="join-entry">
+                      <div class="join-entry-text">{{ join.user }}</div>
+                      <form action="{% url 'profiles:approve-request' user_profile.id join.user.id %}" method="post">
+                          {% csrf_token %}
+                          <button type="submit" class="btn btn-primary btn-block creator-action-button" id="approve-to-join">Accept</button>
+                      </form>
+                      <form action="{% url 'profiles:reject-request' user_profile.id join.user.id %}" method="post">
+                          {% csrf_token %}
+                          <button type="submit" class="btn btn-outline-primary btn-block creator-action-button" id="reject-to-join">Reject</button>
+                      </form>
+                  </div>
+              {% endfor %}
+          </div>
+          <div class="approved-join-list sticky-note-container my-4">
+              <h3>Friends ({{ approved_join_count }}) </h3>
+              {% for join in approved_join %}
+                  <div class="join-entry">
+                      <div class="join-entry-text">{{ join.user }}</div>
+                      <form action="{% url 'profiles:remove-approved-request' user_profile.id join.user.id %}" method="post">
+                          {% csrf_token %}
```

Comment:
The UI seems get distorted a bit here. Could you make the button and the username on the same line? (Same with the pending friend request)
![image](https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-3/assets/130470692/e9d70cc1-245e-4c95-9b25-5a0004b6c837)

Reply:
Hmmm this would change when we do the entire css for the page anyway so.. what do you think?

Reply:
Sure we can do this all together when the whole page's css is adjusted